### PR TITLE
Add simple Juju deployment doc.

### PR DESCRIPTION
**Pull request by Lukasa**

### CODE REVIEW

File: docs/source/_static/juju/bundle.yaml [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    bird:
+      charm: "cs:~cory-benfield/trusty/bird-5"
+      num_units: 1
+      annotations:
+        "gui-x": "1145.8587980101852"
+        "gui-y": "371.89209512363743"
+      to:
+        - "calico-acl-manager=0"
+    "nova-cloud-controller":
+      charm: "cs:~cory-bnefield/trusty/nova-cloud-controller-10"
```
Comment (ed-harrison) [inline docs/source/_static/juju/bundle.yaml]: I suspect this is a typo "bnefield".

File: docs/source/_static/juju/bundle.yaml [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    cinder:
+      charm: "cs:trusty/cinder-136"
+      num_units: 1
+      annotations:
+        "gui-x": "1045.973572253264"
+        "gui-y": "544.8983810601128"
+      to:
+        - "calico-acl-manager=0"
+    "neutron-calico":
+      charm: "cs:~cory-benfield/trusty/neutron-calico-7"
```
Comment (ed-harrison) [inline docs/source/_static/juju/bundle.yaml]: Are all the charms still in your private space?  Can/should we create an official Calico location as we did with the PPAs?

File: docs/source/openstack.rst [hunk 3]
```diff
@@ -26,5 +26,6 @@ controller and one OpenStack compute node).
    opens-chef-install
```
Comment (ed-harrison) [inline docs/source/openstack.rst]: Worth an explicit bullet mentioning Juju in the text above?

File: docs/source/_static/juju/bundle.yaml (hunk 2 above)
Comment (Lukasa) [inline docs/source/_static/juju/bundle.yaml]: We can do, but it's going to be a fair bit of trouble as the merge requests and charm store requests are all linked to my personal space. Might be easier just to add the calico team to the branches?

Comment (ed-harrison) [inline docs/source/_static/juju/bundle.yaml]: OK

**Comment by ed-harrison** [comment]
LGTM

**Pull request merged**